### ISSUE TITLE
Use DocC if open-sourced

### ISSUE THREAD

**Issue by mikolasstuchlik** [rhx/SwiftGtk#52]
An official Apple account mentioned, that DocC might get open-sourced someday.

If it does, we should use it instead of 3rd party library.

We should also update gir2swift so take advantage of the new grammar.

https://developer.apple.com/forums/thread/682023

**Comment by rhx** [comment]
Yes, this is definitely something that would be nice.  Apart from not being open source (yet), another potential stumbling block is that DocC is not compatible with GitHub Pages at the moment.

**Comment by wannafedor4** [comment]
@rhx https://www.swift.org/blog/swift-docc/
:)

**Comment by rhx** [comment]
Yes, I'm transitioning things over on the development branch (not yet ready to merge, though).
